app/src/js/__global.js

```diff
@@ -41,13 +41,13 @@ function showElementById (id) {
 }
 
 function changeFieldSecurity (fieldId) {
-  var security = document.getElementById(fieldId).style.webkitTextSecurity;
+  var security = document.getElementById(fieldId).getAttribute('type');
```

Comment:
since this is no longer a security attribute could we rename this?

Reply:
changed :)